config/isrWrap.py

```diff
@@ -0,0 +1,9 @@
+"""
+Test-specific overrides for IsrWrapperTask
+"""
+import os.path
+from lsst.utils import getPackageDir
+
+ObsConfigDir = os.path.join(getPackageDir("obs_test"), "config")
```

Comment:
Convention is to start variable names that do not refer to types with lowercase.

Reply:
Noted elsewhere, and uniformly fixed across all packages.